spec/requests/topics_controller_spec.rb

```diff
@@ -4052,15 +4052,10 @@ def extract_post_stream
             messages =
               MessageBus.track_publish do
                 put "/topics/reset-new.json", params: { category_id: private_category.id }
+                expect(response.status).to eq(200)
               end
-            expect(response.status).to eq(200)
-            expect(messages.size).to eq(1)
-            expect(messages[0].channel).to eq(TopicTrackingState.unread_channel_key(user.id))
-            expect(messages[0].user_ids).to eq([user.id])
-            expect(messages[0].data["message_type"]).to eq(
-              TopicTrackingState::DISMISS_NEW_MESSAGE_TYPE,
-            )
-            expect(messages[0].data["payload"]["topic_ids"]).to eq([])
```

Comment:
Remove these assertions because we shouldn't publish a message when the payload is empty.